components/expand-collapse/expand-collapse-content.js

```diff
@@ -45,17 +46,29 @@ class ExpandCollapseContent extends LitElement {
 			.d2l-expand-collapse-content-container {
 				display: none;
 				overflow: hidden;
-				transition: height 400ms cubic-bezier(0, 0.7, 0.5, 1);
+				transition:
+					height var(--d2l-expand-collapse-content-transition-duration) cubic-bezier(0, 0.7, 0.5, 1),
+					opacity calc(var(--d2l-expand-collapse-content-transition-duration) * 2) cubic-bezier(0, 0.7, 0.5, 1);
```

Comment:
I'm not certain this change (to the duration) is working the way we expect on collapse. The state of the component is controlled via the `transitionend` event, and the handler will be dispatched for each property that changes. When the first property transition ends (for `height`), the `display` is set to `none` because it thinks the transition is complete, and I think the user would not see the complete opacity transition. Not sure what happens to the opacity `transitionend` event in that case - it may never get dispatched, or maybe it would. For the expand case, the second event would certainly be dispatched 400ms later. 

I think the code is trying to handle the case of quickly toggling the state while transitioning, but it seems like the extra later event could throw off the timing in such scenarios because the second event (from the first toggle) could cause the state to change early (during the second toggle). I don't know for certain this is a problem, but currently, transitioning more than one property with different durations seems dicey. Are we sure we want that? If so, checking `e.propertyName === 'height'` in the handler may be prudent.

If we do feel we need some adjustment to the opacity transition, perhaps it is the timing function we should be changing rather than the duration.